week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,140 @@
+//created new variable by selecting the different ID's using document.getElementById("blueBtn");
+// I assigned the html tag (by using the document.getElementByID) to a new variable called blue button
+//https://www.w3schools.com/jsref/prop_element_children.asp  - 
+let blueBtn = document.getElementById("blueBtn");
+let orangeBtn = document.getElementById("orangeBtn");
+let greenBtn = document.getElementById("greenBtn");
+let jumbotron = document.getElementsByClassName("jumbotron")[0];
+let buttons = document.querySelector(".buttons");
+//var a = b 
+let childrenButtons = buttons.children; // created a variable called childrenButtons (line 58) and the value I am assigning is all the children of the class buttons on line 59 and 60. Which are the next two lines just after html buttons tag.
+let donateBikeBtn = childrenButtons[0]; //donateBike button is being a assigned the as the first child of buttons = [0]
+let volunteerBtn = childrenButtons[1]; //donateBike button is being a assigned the as the second child of buttons = [1]
+
+//console.log(childrenButtons);
+//so on line 16, i called blueBtn variable and added an eventListner which is a function that allow the click event. So inside the function i am calling
+blueBtn.addEventListener("click", function () {
+    jumbotron.style.background = "#588fbd";
+    donateBikeBtn.style.background = "#ffa500";
+    volunteerBtn.style.background = "black";
+    volunteerBtn.style.color = "white";
+    console.log("hello");
+});
+
+//another version of a named function from line 24 - line 31
```

Comment:
It'd be great if you could remove unused comments before you submit your homework so this is a little easier to read ☺